client/src/components/User/Suggestions/Suggestions.js

```diff
@@ -103,7 +105,7 @@ export default class Suggestions extends React.Component {
             {role.toUpperCase() === Role.PARTNER && <>
                 <h6 className="mt-4">Bulk New Monument or Memorial Suggestions</h6>
                 {(!suggestions.bulk || suggestions.bulk.length === 0) && !error && <>
-                    You don't have any {type} bulk new monument or memorial suggestions yet. You can suggest bulk
+                    You don't have any {type} bulk new monument or memorial suggestions yet. You can {createText} bulk
```

Comment:
Since this can only be viewed by partners right now, not a necessary change, but doesn't really matter.